sunpy/net/base_client.py

```diff
@@ -203,7 +203,7 @@ def _print_client(client, html=False):
     if html:
         lines = [f"<p>{line}</p>" for line in lines]
     lines.extend(t.pformat_all(show_dtype=False, max_width=width, align="<", html=html))
```

Comment:
There is an argument to `pformat_all` `max_lines=` which we should use instead of truncating the results: https://docs.astropy.org/en/latest/api/astropy.table.Table.html#astropy.table.Table.pformat_all

Reply:
Hi, @Cadair I have made the change. The latest commit used pformat_all for limiting rows.